airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -865,3 +868,85 @@ def _parse_s3_config(config_file_name: str, config_format: str | None = "boto",
         config_format=config_format,
         profile=profile,
     )
+
+
+class AwsBaseAsyncHook(AwsBaseHook):
```

Comment:
need to remove the changes for base async hook from this PR as that will be contributed from #28850

Reply:
removed